ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/ChatText.qml

```diff
@@ -6,7 +6,7 @@ import QtGraphicalEffects 1.0
 Item {
     property bool longChatText: true
     property bool veryLongChatText: chatsModel.plainText(message).length >
-                                    (appSettings.compactMode ? Constants.limitLongChatTextCompactMode : Constants.limitLongChatText)
+                                    (appSettings.usesCompactMode ? Constants.limitLongChatTextusesCompactMode : Constants.limitLongChatText)
```

Comment:
I think your "Replace all" didn't have good enough constraints haha. You changed the Constant too by accident